wsmaster/che-core-api-factory-bitbucket-server/src/main/java/org/eclipse/che/api/factory/server/bitbucket/BitbucketServerPersonalAccessTokenFetcher.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) 2012-2018 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.che.api.factory.server.bitbucket;
+
+import org.eclipse.che.api.factory.server.scm.PersonalAccessToken;
+import org.eclipse.che.api.factory.server.scm.PersonalAccessTokenFetcher;
+
+public class BitbucketServerPersonalAccessTokenFetcher implements PersonalAccessTokenFetcher {
+  @Override
+  public PersonalAccessToken fetchPersonalAccessToken(String cheUserId, String scmServerUrl) {
+    return null;
```

Comment:
?

Reply:
yes if this concrete PersonalAccessTokenFetcher implementation can't fech it for the given url.
I should better document that - true.